packageDependencies.plist

```diff
@@ -7,7 +7,7 @@
 	<key>awsCRTSwiftVersion</key>
 	<string>0.4.0</string>
 	<key>clientRuntimeBranch</key>
-	<string>main</string>
+	<string>feat/waiters</string>
```

Comment:
This branch should be reverted to `main`:
- after the `feat/waiters` branch in `smithy-swift` merges
- before this branch merges

Reply:
Done